src/propertySelector/tpl/label-text.tpl

```diff
@@ -1 +1 @@
-<span class="property-description-label">{{{text}}}{{#if alias}}/{{/if}}</span>
+<span class="property-description-label">{{{text}}}{{#if alias}} /{{/if}}</span>
```

Comment:
Just a UX question (to be refined if we have time). Now we show `null` alias in case the property is duplicated and does not contain alias. Will we still show it?

Reply:
Good point. This highlights the fact there is nothing regarding the alias nor the class name in the ACs.
I've asked precision here: https://oatsa.slack.com/archives/C026S0SHXHN/p1664265665193939

Reply:
thanks! I saw we had a decision there, so I am closing the thread :)